docs/release.md

```diff
@@ -25,31 +25,34 @@ It is possible that a core approver isn't a contrib approver. In that case, the
 
 ## Releasing opentelemetry-collector
 
+1. Check if there are any open pull requests that are [ready to merge](https://github.com/open-telemetry/opentelemetry-collector/pulls?q=is:open+is:pr+label:ready-to-merge).
```

Comment:
I believe these steps are supposed to be ordered, so doing `1 -> 2 -> 3` makes more sense to me.

Reply:
Reverted the numbering change.